### ISSUE TITLE
IL emulator

### ISSUE THREAD

**Issue by psifertex** [Vector35/binaryninja-api#1212]

**Comment by insuyun** [comment]
Hi. Is there any progress on this?

**Comment by psifertex** [comment]
Nope, this is not currently in development and we have no future details at this time.  There have been several other plugins you might be interested in though related to this: 

https://www.kryptoslogic.com/blog/2020/12/automated-string-de-gobfuscation/
(plugin at: https://github.com/kryptoslogic/binja_degobfuscate)
https://github.com/joshwatson/emilator
